engine/src/main/java/com/sandboni/core/engine/sta/operation/GraphOperations.java

```diff
@@ -44,6 +45,10 @@ public GraphOperations(Graph<Vertex, Edge> graph, Context context) {
     public Set<TestVertex> getAllTests() {
         return allTestsSupplier.get();
     }
+
+    public Set<TestVertex> getAllExternalTests(){
```

Comment:
Same as before, why not use Unit and Cucumber?

Reply:
Fixed